robotpy_build/hooks_datacfg.py

```diff
@@ -17,6 +17,9 @@ class Config:
 class ParamData(Model):
     """Various ways to modify parameters"""
 
+    # Rename parameter
+    name: Optional[str] = None
```

Comment:
Aren't you missing a piece?

Reply:
Also, for consistency, maybe `rename`?

Reply:
Nope - everything uses `name` already, and we just update the param dict with this data.